packages/firebase_vertexai/firebase_vertexai/lib/src/vertex_api.dart

```diff
@@ -213,21 +213,34 @@ final class Candidate {
 /// classification is included here.
 final class SafetyRating {
   /// Constructor
-  SafetyRating(this.category, this.probability);
+  SafetyRating(this.category, this.probability,
+      {this.probabilityScore, this.blocked, this.severity, this.severityScore});
 
   /// The category for this rating.
   final HarmCategory category;
 
   /// The probability of harm for this content.
   final HarmProbability probability;
+
+  /// The score for harm probability
+  final double? probabilityScore;
+
+  /// Whether it's blocked
+  final bool? blocked;
```

Comment:
It would be more idiomatic Dart to call this `isBlocked` or `wasBlocked`. My general preference with naming is to translate to the language idioms from the API idioms as close to the boundary as possible.

Reply:
updated